modules/augment/include/opencv2/augment/rotate.hpp

```diff
@@ -0,0 +1,47 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+
+#ifndef OPENCV_AUGMENT_ROTATE_HPP
+#define OPENCV_AUGMENT_ROTATE_HPP
+#include <opencv2/augment/transform.hpp>
+
+namespace cv { namespace augment {
+
+class CV_EXPORTS_W Rotate : public Transform
+{
+public:
+    /* @brief Constructor
+        @param _angleRange the range of angle values to rotate the image
+    */
+    CV_WRAP Rotate(Vec2f angleRange);
+
+    /* @brief Constructor to initialize the rotation transformation with a specific angle
+    */
+    CV_WRAP Rotate(float angle = 0.f);
```

Comment:
I think that we can remove default value because it's kind of useless to have zero rotation.

You also need to specify if angle in degrees or in radians (let's do it in degrees as in [getRotationMatrix2D](https://docs.opencv.org/master/da/d54/group__imgproc__transform.html#gafbbc470ce83812914a70abfb604f4326))